public/app/core/components/UserEdit/Admin/UserStatus.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { getBackendSrv } from '@grafana/runtime';
+import appEvents from 'app/core/app_events';
+import { OrgUser, StoreState } from 'app/types';
+import { toggleUserStatus } from '../state/actions';
+import { Button } from '@grafana/ui';
+
+export interface Props {
+  userId: number;
+  user: OrgUser;
```

Comment:
same comment as above, no need to send in both userId and user here (and no need for redux on this level)